install_files/ansible-base/roles/grsecurity/tasks/paxctl.yml

```diff
@@ -32,5 +33,5 @@
   # The x and e flags are present by default, but can be removed via
   # `paxctl -z <binary>`. TODO: We may want to update the command
   # to set -Cpmxe explicitly. Warrants further testing on hardware.
-  when: "item.stdout != '- PaX flags: -p---m-x-e-- [{{ item.item }}]' or
+  when: "item.stdout != '- PaX flags: -p---m--E--- [{{ item.item }}]' or
```

Comment:
The PaX output doesn't match what we're trying to set, so this task will run every time, breaking idempotence.